go/internal/feast/featurestore.go

```diff
@@ -39,14 +52,11 @@ type GroupedFeaturesPerEntitySet struct {
 	featureViewNames []string
 	// A list of requested featureName if fullFeatureNames = False or a list of featureViewNameAlias__featureName that share this
 	// entity set
-	featureResponseMeta []string
+	aliasedFeatureNames []string
```

Comment:
nit: line 54 above is a stray comment I believe

Reply:
fixed